VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimsHistoryScreen/ClaimsHistoryScreen.test.tsx

```diff
@@ -61,169 +62,150 @@ jest.mock('../../../../api/authorizedServices/getAuthorizedServices', () => {
   }
 })
 
-const activeClaimsAndAppealsList: api.ClaimsAndAppealsList = [
-  {
-    id: '1',
-    type: 'appeal',
-    attributes: {
-      subtype: 'supplementalClaim',
-      completed: false,
-      decisionLetterSent: false,
-      dateFiled: '2020-10-22',
-      updatedAt: '2020-10-28',
-      displayTitle: 'supplemental claim for disability compensation',
-    },
-  },
-  {
-    id: '0',
-    type: 'claim',
-    attributes: {
-      subtype: 'Disability',
-      completed: false,
-      decisionLetterSent: false,
-      dateFiled: '2020-11-13',
-      updatedAt: '2020-11-30',
-      displayTitle: 'Disability',
-    },
-  },
-  {
-    id: '4',
-    type: 'claim',
-    attributes: {
-      subtype: 'Compensation',
-      completed: false,
-      decisionLetterSent: false,
-      dateFiled: '2020-06-11',
-      updatedAt: '2020-12-07',
-      displayTitle: 'Compensation',
-    },
-  },
-]
-
-const mockPagination: ClaimsAndAppealsGetDataMeta = {
-  dataFromStore: false,
-  pagination: {
-    currentPage: 1,
-    perPage: 10,
-    totalEntries: 3,
-  },
-  activeClaimsCount: 0,
-}
-
-const mockPaginationClaimsServiceError: ClaimsAndAppealsGetDataMeta = {
-  ...mockPagination,
-  errors: [
+const mockPayload: ClaimsAndAppealsListPayload = {
+  data: [
     {
-      service: 'claims',
+      id: '0',
+      type: 'appeal',
+      attributes: {
+        subtype: 'supplementalClaim',
+        completed: false,
+        decisionLetterSent: false,
+        dateFiled: '2020-10-22',
+        updatedAt: '2020-10-28',
+        displayTitle: 'supplemental claim for disability compensation',
+      },
     },
-  ],
-}
-
-const mockPaginationAppealsServiceError: ClaimsAndAppealsGetDataMeta = {
-  ...mockPagination,
-  errors: [
     {
-      service: 'appeals',
+      id: '2',
+      type: 'claim',
+      attributes: {
+        subtype: 'Compensation',
+        completed: false,
+        decisionLetterSent: false,
+        dateFiled: '2020-10-22',
+        updatedAt: '2020-10-30',
+        displayTitle: 'Compensation',
+      },
     },
   ],
-}
-
-const mockPaginationAppealsClaimsServiceError: ClaimsAndAppealsGetDataMeta = {
-  ...mockPagination,
-  errors: [
-    {
-      service: 'claims',
+  meta: {
+    pagination: {
+      currentPage: 1,
+      perPage: 10,
+      totalEntries: 3,
     },
-    {
-      service: 'appeals',
-    },
-  ],
+  },
 }
 
 context('ClaimsHistoryScreen', () => {
-  const initializeTestInstance = () => {
-    render(<ClaimsHistoryScreen {...mockNavProps()} />)
+  const initializeTestInstance = (errors: Array<ClaimsAndAppealsGetDataMetaError>) => {
```

Comment:
The `errors` parameter can be optional, so that you wouldn't need to pass an empty array for the instances that don't have errors

Reply:
updated